src/views/SpaceProposal.vue

```diff
@@ -278,7 +278,7 @@ const truncateMarkdownBody = computed(() => {
                 }"
               >
                 <div class="flex items-center">
-                  <Token :space="space" size="28" />
+                  <BaseToken :space="space" size="28" />
```

Comment:
"Token" name itself is misleading, this is something we could also change

Reply:
Yeah I agree, maybe this could just be a composable?

Reply:
A composable to return an image? Not sure how you mean it. I'd rather use a component for it. We could have a component "Avatar" where we can add type="space" or type="user" or a component "SpaceAvatar"

Reply:
Nvm it doesn't make sense, let's do a component.